low-level/framework/messaging/egress_processor.py

```diff
@@ -81,11 +81,17 @@ def initialize(self, conf_reader, msgQlist, product):
         self._request_shutdown = False
 
         self._read_config()
-        self._producer = MessageProducer(producer_id=self._producer_id,
-                                         message_type=self._message_type,
-                                         method=self._method)
+        self.create_MsgProducer_obj()
```

Comment:
can we have one function somewhere, both for consumer & producer, which can used from all places where ever needed

Reply:
Yes, we can have, where we can define this??
should I create separate class inside - low-level/framework/messaging/create_msgbus_instance.py
`class MessageBusInstance:
      def create_MsgProducer_obj():
      
      def create_MsgConsumer_obj():
`

Reply:
as discussed, no. New class just to maintain a function to create an object, doesnt make sense. So let it as it is today